FinniversKit/Sources/Extensions/UIKit/UITraitCollectionExtensions.swift

```diff
@@ -1,7 +1,7 @@
 import UIKit
 
 extension UIView {
-    var isHorizontalSizeClassRegular: Bool { traitCollection.horizontalSizeClass == .regular }
+    public var isHorizontalSizeClassRegular: Bool { traitCollection.horizontalSizeClass == .regular }
```

Comment:
if i'm not mistaken this code may be already be copied into FinnUI and the iOS app, maybe it will trigger a warning

Reply:
I have now copied this file to FinnUI as well, and kept it internal in both FinniversKit and FinnUI, to avoid any potential warnings 👍  (I didn't find one, but I guess this should have triggered a warning..)